drv/gimlet-seq-server/src/main.rs

```diff
@@ -111,11 +121,16 @@ enum Trace {
     SpdBankAbsent(u8),
     SpdAbsent(u8, u8, u8),
     SpdDimmsFound(usize),
-    I2cFault {
-        retries_remaining: u8,
+    I2cError {
+        txn: I2cTxn,
```

Comment:
One deficiency of the counters crate is that we cannot currently place `#[count(children)]` on multiple fields to generate counts of each unique pair of that field, it would be nice to do that so that we can count the number of errors by transaction type. I might hack that together and we can come back and add it here.